add eip-6538 (#16)
Nerolation/EIP-ERC-721-Stealth-Addresses: eip-5564.md

st:eth:0x<spendingKey><viewingKey>
``` 

> Stealth meta-addresses are intended to be autonomously managed by the user and/or registered within a publicly available `Registry` contract, as delineated in [ERC-6538](./eip-6538). This provides users with a centralized location for identifying stealth meta-addresses associated with other individuals while simultaneously enabling recipients to express their openness to engage via stealth addresses.

Reviewer: nit, otherwise lgtm! 
```suggestion
Stealth meta-addresses may be managed by the user and/or registered within a publicly available `Registry` contract, as delineated in [ERC-6538](./eip-6538). This provides users with a centralized location for identifying stealth meta-addresses associated with other individuals while simultaneously enabling recipients to express their openness to engage via stealth addresses.
```
